### DISCUSSION TITLE
About dp model of “The Phase Diagram of a Deep Potential Water Model”

### DISCUSSION THREAD

**Discussion by zhao-w-en** [deepmodeling/dpgen#699] [Q&A] [answered]
Hello developers,
 I recently read article titled “The Phase Diagram of a Deep Potential Water Model” which was published in PRL and we want to do some research following your work. However, it seems that we only found the dataset on DPLibrary, which would take much time on training. I wonder if there is a **constructed DP model** on Github. If so, could you please share this link with me? I really can't find it.
Thanks in adcvance and I'm sorry for your inconvenience.

Your sincerely,
Wen zhao

**Comment by AnguseZhang** [comment, marked as answer]
[graph.pb.zip](https://github.com/deepmodeling/dpgen/files/8472006/graph.pb.zip)
Have a try.

**Reply by zhao-w-en** [reply]
Thank u very much
